### PR TITLE
Added SimpleRadial camera model to colmap.cpp

### PR DESCRIPTION
**Pull request by Jonathan-Gore**
Added the SimpleRadial camera model to colmap.cpp from https://github.com/colmap/colmap/blob/0ea2d5ceee1360bba427b2ef61f1351e59a46f91/src/colmap/sensor/models.h#L235

Copied syntax found in SimplePinhole, copying fy=fx for the models with only one focal length parameter, like SimpleRadial.

I'm new to c++ so I'm not 100% sure how to test the code.

### CODE REVIEW

File: colmap.cpp [hunk 1]
```diff
@@ -53,6 +53,12 @@ InputData inputDataFromColmap(const std::string &projectRoot){
             cam->fy = readBinary<double>(camf);
             cam->cx = readBinary<double>(camf);
             cam->cy = readBinary<double>(camf);
+        }else if (model == SimpleRadial){
+            cam->fx = readBinary<double>(camf);
+            cam->fy = cam->fx;
+            cam->cx = readBinary<double>(camf);
+            cam->cy = readBinary<double>(camf);
+            cam->k = readBinary<double>(camf);
```
Comment (pierotofy) [inline colmap.cpp:61]: `k` is not a valid parameter. See https://github.com/pierotofy/OpenSplat/blob/main/input_data.hpp#L12

**Comment by pierotofy** [comment]
Thanks for the PR @Jonathan-Gore ! See the "Build" section from the README to compile and test the changes.

File: colmap.cpp (hunk 1 above)
Comment (Jonathan-Gore) [reply colmap.cpp:61]: Understood, I was copying verbatim from the colmap parameter list. Would 'k1' be a reasonable substitute in this situation for what I designated as 'k'? I don't fully understand what the other parameters represent past the 'fx' and 'fy' focal length parameters.

**Comment by Jonathan-Gore** [comment]
Updated k -> k1 and successfully built the application! Ran the newly built .exe on the banana dataset with both PINHOLE and SIMPLE_RADIAL camera models and both were successful!
[simple_radial_splat.zip](https://github.com/pierotofy/OpenSplat/files/14953153/simple_radial_splat.zip)

**Comment by pierotofy** [comment]
Fantastic, thanks for testing and the useful addition! 🙏

**Pull request merged**